source/backend/api/Models/Concepts/Project/ProjectModel.cs

```diff
@@ -48,6 +50,16 @@ public class ProjectModel : BaseModel
         /// get/set - Project notes.
         /// </summary>
         public string Note { get; set; }
+
+        /// <summary>
+        /// get/set - Last Updated Date.
+        /// </summary>
+        public DateTime AppLastUpdateTimestamp { get; set; }
+
+        /// <summary>
+        /// get/set - Last Updated by.
+        /// </summary>
+        public string AppLastUpdateUserid { get; set; }
```

Comment:
These fields should come in the base Model already

Reply:
It didn't. Replaced with 'BaseAppModel' which does.

Reply:
Updated